sensitivity-analysis-api/src/test/java/com/powsybl/sensitivity/factors/BusVoltagePerTargetVTest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sensitivity.factors;
+
+import com.powsybl.sensitivity.factors.functions.BusVoltage;
+import com.powsybl.sensitivity.factors.variables.TargetV;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+/**
+ * @author Anne Tilloy {@literal <anne.tilloy at rte-france.com>}
+ */
+public class BusVoltagePerTargetVTest {
```

Comment:
For consistency sake, we should maybe add a JSON test for that new factor in the `json` package.